GroovyRP/Program.cs

```diff
@@ -1,191 +1,93 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using DiscordRPC;
-using CSCore;
-using CSCore.CoreAudioAPI;
-using TagLib;
-using CsvHelper;
-using System.IO;
-using System.Globalization;
-using System.ComponentModel;
+using DiscordRPC.Message;
 
 namespace GroovyRP
 {
     class Program
     {
-        public static DiscordRpcClient client;
+        private const string appDetails = "GroovyRP\nOrignal Creator: https://github.com/dsdude123/GroovyRP \nThis Version: https://github.com/DraxCodes/GroovyRP\n\n";
+        private static readonly DiscordRpcClient _client = new DiscordRpcClient("737646410354130955", autoEvents: false);
```

Comment:
Please send me the asset changes so that they can be applied to the existing client ID. Either can post them here or send details to csmediaseattle@gmail.com

Reply:
WIll post it as a seperate comment.